samples/sample-manifest.json

```diff
@@ -20,13 +20,24 @@
             "MaxActivations": 1,
             "Parameters": {
                 "Files": {
-                    "/tmp/defg.txt": "foo",
-                    "/tmp/jkl.mno": "bar"
+                    "/tmp/abc.txt": "helloworld",
+                    "/tmp/helloworld.sh": "#!/usr/bin/env bash\necho {{ raw .Secrets.hello }} {{ raw .Secrets.hello }}",
+                    "/tmp/defg.txt": {
+                        "encoding": "string",
+                        "data": "foo"
+                    },
+                    "/tmp/jkl.mno": {
+                        "encoding": "base64",
+                        "data": "YmFy"
+                    },
+                    "/tmp/enc.key": {
+                        "encoding": "bytes",
+                        "data": "{{ raw .Secrets.symmetric_key_shared }}"
                 },
                 "Env": {
                     "IS_FIRST": "true",
                     "SEAL_KEY": "{{ hex .Marblerun.SealKey }}",
-                    "TEST_SECRET_SYMMETRIC_KEY": "{{ raw .Secrets.symmetric_key_shared }}",
+                    "API_KEY": "{{ raw .Secrets.api_key }}",
```

Comment:
What about a new modifier like `utf-8` in addition to `raw`?

Reply:
I renamed `raw` to `string` does that make more sense?
I'm open to a different name though.
We're going to add `utf-8` or similar modifiers in a separate step.

Reply:
For people familiar with Go typing system this might work, as we quiet literally just convert and write the bytes of the secret as a string.
The difference between `string` and `utf-8`, and what `string` actually does, will need to be well documented in the docs to avoid mistakes made by the user.
Especially to prevent accidental writing of binary data through the use `string`.